tests/vm_util_test.py

```diff
@@ -17,10 +17,56 @@
 import unittest
 
 import mock
+import os
```

Comment:
I tried coming up with some mock/patch to capture the execvp/execvpe function calls, but ran into a ton of problems. Took a platform-specific approach, since it executes /usr/bin/env and looks at the stdout.

What are you thoughts? I feel like I should just drop these tests, since all I'm doing is passing a parameter through the function to pass to Popen(), and not adding any logic around it.